docs/overview/excel.md

```diff
@@ -1,15 +1,17 @@
 ---
 title: 'Office Scripts in Excel on the web'
 description: 'A brief introduction to the Action Recorder and Code Editor for Office Scripts.'
-ms.date: 01/21/2020
+ms.date: 02/24/2020
 localization_priority: Priority
 ---
 
 # Office Scripts in Excel on the web (preview)
 
 Office Scripts in Excel on the web let you automate your day-to-day tasks. You can record your Excel actions with the Action Recorder, which creates a script. You can also create and edit scripts with the Code Editor. This series of documents teaches you how to use these tools. You'll be introduced to the Action Recorder and see how to record your frequent Excel actions. You'll also learn how to make or update your own scripts with the Code Editor.
 
-![A gif showing a script being recorded and replayed on different worksheets](../images/teaser-animation.gif)
+<br>
+
+> [!VIDEO https://www.microsoft.com/videoplayer/embed/RE4qdFF]
```

Comment:
also I don't think you need the >

Reply:
I'm following [this reference page](https://review.docs.microsoft.com/en-us/office-authoring-guide/add-a-video-to-article?branch=master), which seems to indicate I do. I'll test.